test/routes/mail/index.spec.js

```diff
@@ -0,0 +1,70 @@
+import nock from 'nock';
+import assert from 'power-assert';
+import request from 'supertest';
+import app from 'app.js';
+
+const SLACK_POST = process.env.KINDLIZED_SLACK;
+
+describe('/routes/mail', ()=> {
+  let response;
+  const slackResponse = [
+    { msg: { html: 'メールの本文' } },
+  ];
+  before((done)=> {
+    nock(SLACK_POST)
+      .post('')
+      .reply(200, 'Slack relay is completed.');
+
+    request(app)
+      .post('/mail')
+      .send({
+        mandrill_events: JSON.stringify(slackResponse),
+      })
+      .end((err, ret)=> {
+        if (err) {
+          response = err;
+        }
+        response = ret;
+        done();
+      });
+  });
+
+  it('MandrillからのメールPOSTを中継できる', ()=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).